Bottom: 0e30d09ac5bdddb4cdcf0d2629afbc6e8f70f6c9
Top:    856be74e30e660457cd6da48c60dc541a5bfe929
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 10:39:38 +0000

Refresh of remove-spaces-from-step-output

---

diff --git a/src/hid/step/step_writer.c b/src/hid/step/step_writer.c
index 37afac9..d64e062 100644
--- a/src/hid/step/step_writer.c
+++ b/src/hid/step/step_writer.c
@@ -297,7 +297,7 @@ step_face_bound (step_file *file, char *name, step_id bound, bool orientation)
 step_id
 step_face_outer_bound (step_file *file, char *name, step_id bound, bool orientation)
 {
-  fprintf (file->f, "#%i = FACE_OUTER_BOUND ( '%s', #%i, %s ) ;\n",
+  fprintf (file->f, "#%i=FACE_OUTER_BOUND('%s',#%i,%s);",
                     file->next_id, name, bound, step_bool (orientation));
 
   return file->next_id++;
